masterlist.yaml

```diff
@@ -395,29 +397,41 @@ plugins:
         content:
           - lang: en
             text: '"ccqdrsse001-survivalmode.esl" has a [Known bug](https://www.nexusmods.com/skyrimspecialedition/articles/22/) which is incompatible with "Ordinator - Perks of Skyrim.esp". Consider using the [Unofficial skyrim survival patch](https://www.nexusmods.com/skyrimspecialedition/mods/12655/) to fix this bug.'
+### Unofficial Patches
   - name: 'Unofficial Skyrim Special Edition Patch.esp'
     global_priority: -120
+    after:
+      - 'Lanterns Of Skyrim - All In One - Main.esm'
     clean:
+      # Version 4.1.2alpha
       - crc: 0xAE28DBE3
         util: SSEEdit v3.2.1
-    after:
-      - 'Lanterns Of Skyrim - All In One - Main.esm'
   - name: 'unofficial skyrim survival patch.esp'
+    tag:
+      - Graphics
     clean: 
+      # version 2.9
       - crc: 0xF7B1A347
         util: SSEEdit v3.2.1
-    tag:
-      - Graphics
+### Other Plugins
   - name: 'BSAssets.esm'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/10917/']
```

Comment:
I know the URL are not strictly required, but it does make it easier for maintenance.

Reply:
Yep 👍